### PR TITLE
[MM-28214] Billing Summary - Last Invoice View

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
Billing Summary - Last Invoice view.
Shows the last invoice billed to the installation.

Will need UX and QA review after merge.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28214

#### Related Pull Requests
- Has redux changes (https://github.com/mattermost/mattermost-redux/pull/1271)

### CODE REVIEW

File: components/admin_console/billing/billing_summary.tsx [hunk 1]
```diff
@@ -45,9 +53,210 @@ const noBillingHistory = (
...
+                            <FormattedNumber
+                                value={(charge.total / 100.0)}
+                                // eslint-disable-next-line react/style-prop-object
+                                style='currency'
+                                currency='USD'
+                            />
+                        </div>
+                    </div>
+                ))}
+                {invoice.tax &&
```
Comment (marianunez) [inline components/admin_console/billing/billing_summary.tsx:195]: If there are no taxes, I believe we should still show the line but with 0 as the value to be consistent with how it is presented in the actual invoice

**Review by marianunez — CHANGES_REQUESTED** [review]
LGTM, just a change to always show Taxes line

File: components/admin_console/billing/billing_summary.tsx (hunk 1 above)
Comment (devinbinnie) [reply components/admin_console/billing/billing_summary.tsx:195]: I feel like it's going to waste screen real estate for many users, especially in the beginning where apparently we're only charging tax for a subset of customers. I don't think people need to see that they have a 0 charge for something that's not relevant to them.

Comment (marianunez) [reply components/admin_console/billing/billing_summary.tsx:195]: @matthewbirtch any thoughts on this whether to completely hide the Tax line if it's zero?

Comment (matthewbirtch) [reply components/admin_console/billing/billing_summary.tsx:195]: I think it's fine to hide the tax line if it's zero. This is just a summary view anyway, so it's reasonable that we don't show lines that are irrelevant. My 2cents, but not sure if it's super important from a finance perspective

**Review by marianunez — APPROVED** [review]

**Comment by matthewbirtch** [comment]
@devinbinnie I just attempted to review this with the test cloud server, but the test server does not have the billing and account section in the system console

**Comment by devinbinnie** [comment]
> @devinbinnie I just attempted to review this with the test cloud server, but the test server does not have the billing and account section in the system console

@matthewbirtch We can't test this on a SpinWick, as there's nothing hooked up to the CWS. So it's going to have to happen later.

File: components/admin_console/billing/billing_summary.tsx [hunk 2]
```diff
@@ -45,9 +53,210 @@ const noBillingHistory = (
...
+        body = (
+            <div className='BillingSummary__lastInvoice'>
+                <div className='BillingSummary__lastInvoice-header'>
+                    <div className='BillingSummary__lastInvoice-headerTitle'>
+                        <FormattedMessage
+                            id='admin.billing.subscriptions.billing_summary.lastInvoice.title'
+                            defaultMessage='Last Invoice'
+                        />
+                    </div>
+                    {invoice.status === 'paid' &&
```
Comment (ethervoid) [inline components/admin_console/billing/billing_summary.tsx:80]: There is another state that could appear "open" so maybe we want to have it into account

**Review by ethervoid — COMMENTED** [review]
Just a minor comment

File: components/admin_console/billing/billing_summary.tsx (hunk 2 above)
Comment (devinbinnie) [reply components/admin_console/billing/billing_summary.tsx:80]: That's news to me...Is that like a Pending state?

Comment (ethervoid) [reply components/admin_console/billing/billing_summary.tsx:80]: Yep kind of. You can see [here](https://github.com/mattermost/customer-web-server/pull/257/files/f22143573bae995072b9cfb3cf07ea15dda7e99d#diff-e519a9713aedfa590f5a709c33f23d9d7a9dc9c20e95b0a1236870db46abff0aR124) is like "we don't know the state yet so we mark it as open"

Comment (devinbinnie) [reply components/admin_console/billing/billing_summary.tsx:80]: Ahh okay so it's more of a default case. I do the same thing on the Billing History screen so I'll mimic it here.

**Review by ethervoid — APPROVED** [review]
LGTM! great work!

**Comment by matthewbirtch** [comment]
> @matthewbirtch We can't test this on a SpinWick, as there's nothing hooked up to the CWS. So it's going to have to happen later.

Ok, I'll remove myself as a reviewer for now then. You can add me back when it's ready

**Pull request merged**